src/layout/mod.rs

```diff
@@ -4315,12 +4357,18 @@ mod tests {
         };
 
         let mon = monitors.into_iter().next().unwrap();
+        let expected_workspace_offset = if mon.options.empty_workspace_above_first {
```

Comment:
Why this check? The test is done with default options.

Reply:
Ups, when testing I temporarily changed the Default, this is a remnant of that. I'll refactor it to it's own test.